evaluation/examples/linux-cpp/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu
```

Comment:
I don't think so - the participants can put whatever they wish inside the image.